specification/trace/semantic_conventions/cloudevents.md

```diff
@@ -37,14 +39,13 @@ In case a CloudEvent is passed to the instrumented library with the Distributed
 
 ### Processing
 
-To trace the processing of one or more CloudEvents, instrumentation SHOULD create a new span.
-If a single event is processed, instrumentation SHOULD use the remote trace context from the Distributed Tracing Extension as a parent or MAY instead add it as a link on the processing span.
-
-If multiple events are processed together, for each event being processed, if the event has the Distributed Tracing Extension populated, instrumentation MUST add a link to the trace context from the extension on the processing span.
+When instrumented library supports processing of a single CloudEvent, instrumentation SHOULD create a new span to trace it. Instrumentation SHOULD set the remote trace context from the Distributed Tracing Extension as a link on the processing span.
 
 **Span name:** `CloudEvents Process <event_type>`
 **Span kind:** CONSUMER
 
+Note: CloudEvents processing does not follow a common pattern. Some CloudEvents-enabled libraries expose handlers for CloudEvents processing (e.g.  [CloudEvents Go SDK](https://github.com/cloudevents/sdk-go#receive-your-first-cloudevent)) and are able to instrument processing calls. In other cases CloudEvents are deserialized or processed in application code and then application is responsible for instrumentation.
```

Comment:
```suggestion
Note: CloudEvents processing does not follow a common pattern. Some CloudEvents SDKs expose handlers for processing (e.g.  [CloudEvents Go SDK](https://github.com/cloudevents/sdk-go#receive-your-first-cloudevent)) and are able to auto-instrument processing calls. In other SDKs, CloudEvents are deserialized or processed in application code. In this case, the instrumentation responsibility falls under the application owner.
```

Reply:
I'm not sure `CloudEvents SDK` is precise enough. I can send CloudEvents using basically anything, not just CloudEvents SDKs.

Reply:
You're right, that seems to be the case for some SDKs. I added some suggestion below.